code_generator/End2End/Tests/BlittableComponentTests.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Diagnostics;
+using System.Linq;
```

Comment:
It looks like you can clean up the imports a bit?

Reply:
Done